integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/mockapis/Elite2Api.groovy

```diff
@@ -512,6 +593,18 @@ class Elite2Api extends WireMockRule {
         .withStatus(200)))
   }
 
+  void stubUncategorisedForWomenYO(bookingId, location) {
```

Comment:
nit picking can you change the name to stubUncategorisedForWomenYOI?  Just makes it clearer.

Reply:
ok will update it